.gitmodules

```diff
@@ -31,3 +31,9 @@
 [submodule "modules/staging"]
 	path = modules/staging
 	url = https://github.com/nanliu/puppet-staging.git
+[submodule "modules/haproxy"]
+	path = modules/haproxy
+	url = https://github.com/puppetlabs/puppetlabs-haproxy
+[submodule "modules/concat"]
```

Comment:
Not sure we need concat either - unless any of your code introduces it (looks like it doesn't) ?